src/table/src/thin_table.rs

```diff
@@ -12,76 +12,61 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use std::any::Any;
+use snafu::ResultExt;
 
-use async_trait::async_trait;
 use common_query::prelude::Expr;
 use common_recordbatch::SendableRecordBatchStream;
 use datatypes::schema::SchemaRef;
-use snafu::ResultExt;
 use store_api::data_source::DataSourceRef;
 use store_api::storage::ScanRequest;
 
 use crate::error::{Result, TablesRecordBatchSnafu};
 use crate::metadata::{FilterPushDownType, TableInfoRef, TableType};
-use crate::Table;
 
 /// The `ThinTable` struct will replace the `Table` trait.
 /// TODO(zhongzc): After completion, perform renaming and documentation work.
 pub struct ThinTable {
```

Comment:
If this PR is good to go, I can rename the struct in this PR or in a follow-up.

Reply:
Great job! I prefer to refactor `crate::thin_table::ThinTable` to `crate::Table` in this PR.

Reply:
Updated. Please take a look.